PyStemmusScope/config_io.py

```diff
@@ -102,8 +102,8 @@ def _update_config_file(nc_file, input_dir, output_dir, config, station_name, ti
     config_file_path = input_dir / f"{station_name}_{timestamp}_config.txt"
     with open(config_file_path, 'w', encoding="utf8") as f:
         for key, value in config.items():
-            if key == "ForcingFileName":
-                update_entry = f"{key}={nc_file}\n"
+            if key == "Location":
+                update_entry = f"{key}={station_name}\n"
```

Comment:
`if key == "Location":` statement is not needed, it is already covered by the statement after `else` in this block.